extensions/api/data-management/asset/src/main/resources/META-INF/services/org.eclipse.dataspaceconnector.spi.system.ServiceExtension

```diff
@@ -0,0 +1,15 @@
+#
+#  Copyright (c) 2022 Diego Gomez
```

Comment:
As far as I know the Copyright should go to ZF, at least that is my latest information.